### DISCUSSION TITLE
"Deprecating Facebook Login support on Android WebViews",Did anyone upgrade successfully?

### DISCUSSION THREAD

**Discussion by CoolFishGames** [airsdk/Adobe-Runtime-Support#1163] [Q&A]
https://developers.facebook.com/docs/facebook-login/android/deprecating-webviews
We try upgrade, bug  still start with webwiew; Is this the case of airsdk?

**Comment by marchbold** [comment]
Are you using our ANE (distriqt)? We have updated successfully, just make sure you correctly set the latest manifest additions, there were some changes to the custom tabs.

**Reply by CoolFishGames** [reply]
> Are you using our ANE (distriqt)? We have updated successfully, just make sure you correctly set the latest manifest additions, there were some changes to the custom tabs.

Sorry, no, we use the open source project(freshplanet), but this project has not been updated, so we are trying to modify it ourselves. Thank you very much for your reply.

**Comment by Klug76** [comment]
Hello! Please, can you show in more detail what changes occurred in the custom tabs part of manifest?

**Reply by marchbold** [reply]
Make sure you have the following additions:

```xml
        <activity android:name="com.facebook.CustomTabMainActivity" />
        <activity
            android:name="com.facebook.CustomTabActivity"
            android:exported="true" >
            <intent-filter>
                <action android:name="android.intent.action.VIEW" />
                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />
                <data
                    android:host="cct.APPLICATION_PACKAGE"
                    android:scheme="fbconnect" />
            </intent-filter>
        </activity>
```

Older versions were slightly different. 

Complete manifest list here: https://docs.airnativeextensions.com/docs/facebookapi/core/add-the-extension#manifest-additions
